requirements.txt

```diff
@@ -1,2 +1,3 @@
-apache-airflow==1.10.4
+pip==20.1
+apache-airflow[crypto,postgres,ssh,gcp,kubernetes]==1.10.10
```

Comment:
some [extra packages](https://airflow.apache.org/docs/stable/installation.html) were missing that we have installed in [the Dockerfile](https://github.com/bcgov/cas-helm/blob/master/docker/cas-airflow/Dockerfile#L18)